src/goto-programs/structured_trace_util.h

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************\
+
+Module: Goto Programs
+
+Author: Diffblue
+
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_PROGRAMS_STRUCTURED_TRACE_UTIL_H
+#define CPROVER_GOTO_PROGRAMS_STRUCTURED_TRACE_UTIL_H
+
+#include "goto_program.h"
+#include <string>
+
+/// There are two kinds of step for location markers - location-only and
+/// loop-head (for locations associated with the first step of a loop).
+enum class default_step_kindt
```

Comment:
Why "default"?

Reply:
Nomenclature taken from the docs which call these default steps :shrug: